index.js

```diff
@@ -60,15 +60,63 @@ renderPlayer(player)
 
 
 /***** Deliverable 1 *****/
-function toggleColor(element) {
-  if (element.style.color === "red") {
-    element.style.color = "black"
+// function toggleColor(element) {
+//   if (element.style.color === "red") {
+//     element.style.color = "black"
+//   } else {
+//     element.style.color = "red"
+//   }
+// }
+let header = document.querySelector("h1#header")
+
+header.addEventListener("click", function(){
+  if (header.style.color === "red") {
+    header.style.color = "black"
   } else {
-    element.style.color = "red"
+    header.style.color = "red"
   }
-}
+})
+
 
 /***** Deliverable 2 *****/
+const button = document.querySelector(".like-button")
 
+button.addEventListener("click", increment)
+
+function increment(){
+  const display = document.querySelector(".likes")
+  const likes = parseInt(display.textContent) + 1
+  display.textContent = `${likes} Likes`
+}
```

Comment:
Looks good! The key here is that you have to *read* some information from the DOM to get the current number of likes, increment it, and then *write* it back to the DOM. Nice work!